### ISSUE TITLE
WakaTime throws an error when trying to enter API Key after initial config

### ISSUE THREAD

**Issue by michaelaflores** [wakatime/WakaTime.novaextension#7]
Noticing that attempting to run the `api key` command to check the API key throws this error:
<img width="379" alt="Screen Shot 2021-03-23 at 3 53 32 PM" src="https://user-images.githubusercontent.com/11711706/112216835-f6439180-8bef-11eb-9568-27c08e34562a.png">

My `.wakatime.cfg` file has:
```
[settings]
api_key = <key>
```

Nova version: 5 Build 245280
macOS: 11.2.3 (20D91)
Device: MacBook Air (M1, 2020)

**Comment by michaelaflores** [comment]
I believe this is perhaps related to WakaTime not reporting any time that I spend in Nova? The debug pane reports:
```
Extension encountered an uncaught exception:
/Users/michaelflores/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 382, Column 0)
Error: Error launching process: /Volumes/Macintosh HD/Users/michaelflores/.wakatime/wakatime-cli/wakatime-cli, The operation couldn’t be completed. Bad CPU type in executable
```

**Comment by alanhamlett** [comment]
Duplicate of #6. Does creating a `~/.wakatime.cfg` file fix it? If it's arch related, try using the Python source version by adding this line to your `~/.waktime.cfg` file:
`standalone = false`

**Comment by michaelaflores** [comment]
@alanhamlett I created this one because it happens _after_ the first run when there is in fact a config file already present (in the description I included the contents of that file). #6 relates to the very first run, where the error is related to a config file not being present yet. 

I will give adding that line a try and report back, thank you!

**Comment by michaelaflores** [comment]
@alanhamlett unfortunately after adding that line, I still get the following debug output 

```
WakaTime[10:07:48.221000] [WakaTime] [DEBUG] Initializing version 2.0.0

WakaTime[10:07:48.532000] [WakaTime] [ERROR] Error: Error launching process: /Volumes/Macintosh HD/Users/michaelflores/.wakatime/wakatime-cli/wakatime-cli, The operation couldn’t be completed. Bad CPU type in executable

WakaTime[10:07:48.533000] [WakaTime] [DEBUG] Finished initializing WakaTime extension.
```

**Comment by alanhamlett** [comment]
Can you try deleting the folder `/Volumes/Macintosh HD/Users/michaelflores/.wakatime/wakatime-cli` and restarting Nova?

**Comment by michaelaflores** [comment]
@alanhamlett when I did that and restarted Nova, I saw debug logs to the effect of "downloading wakatime-cli". Upon restarting Nova thereafter, I see the same logs. Is there some check of the CPU type that assumes an Intel processor? 

```
WakaTime[13:17:51.601000] [WakaTime] [DEBUG] Initializing version 2.0.0

WakaTime[13:17:51.913000] [WakaTime] [ERROR] Error: Error launching process: /Volumes/Macintosh HD/Users/michaelflores/.wakatime/wakatime-cli/wakatime-cli, The operation couldn’t be completed. Bad CPU type in executable

WakaTime[13:17:51.914000] [WakaTime] [DEBUG] Finished initializing WakaTime extension.
```

**Comment by michaelaflores** [comment]
After trying to use 2.0.1 I now get this in the debug panel when starting Nova/trying to use any WakaTime commands 

```
WakaTime[17:19:01.792000] Extension encountered an uncaught exception:
/Users/michaelflores/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 200, Column 0)
ReferenceError: Can't find variable: config
    getSetting@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:200:23
    getApiKey@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:280:25
    promptForApiKey@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:43:27
    file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:21:18
```

Here is what's located at ~:
<img width="267" alt="Screen Shot 2021-03-24 at 5 19 17 PM" src="https://user-images.githubusercontent.com/11711706/112391180-869fd600-8cc5-11eb-9991-b671970232ce.png">

**Comment by michaelaflores** [comment]
I tried uninstalling, deleting .wakatime and .wakatime.cfg, and installing again. After prompting for my API key, I entered it, and the following error showed up in debug panel:
```
WakaTime[17:24:27.922000] Extension encountered an uncaught exception:
/Users/michaelflores/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 235, Column 0)
ReferenceError: Can't find variable: lines
    setSetting@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:235:28
    file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:52:39
```

I tried restarting Nova, and now see:

```
WakaTime[17:25:19.977000] [WakaTime] [ERROR] Error: Error launching process: /Volumes/Macintosh HD/Users/michaelflores/.wakatime/wakatime-cli/wakatime-cli, The operation couldn’t be completed. Bad CPU type in executable
```

Then I am reprompted for my API key, and after entering it, the debug panel shows:
```
WakaTime[17:25:54.989000] Extension encountered an uncaught exception:
/Users/michaelflores/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 235, Column 0)
ReferenceError: Can't find variable: lines
    setSetting@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:235:28
    file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:52:39
```

If I recreate .wakatime.config manually, the error changes to

```
WakaTime[17:27:38.634000] Extension encountered an uncaught exception:
/Users/michaelflores/Library/Application Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js (Line 200, Column 0)
ReferenceError: Can't find variable: config
    getSetting@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:200:23
    isDebugEnabled@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:288:56
    debug@file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:447:24
    file:///Users/michaelflores/Library/Application%20Support/Nova/Extensions/wakatime.timetracker/Scripts/main.js:2:12
```

**Comment by SuEric** [comment]
Having same exact issue than @michaelaflores

**Comment by mikebronner** [comment]
Having this issue as well. `~/.wakatime.cfg` already exists for me as I use it in many other programs.

**Comment by alanhamlett** [comment]
Should be fixed in v2.0.2?

**Comment by mikebronner** [comment]
@alanhamlett That's awesome! Not getting any more errors or crashes when setting API Key. Will monitor over the next few days to watch time tracking :) Thanks for pushing this through!
